landing-page/content/common/format/spec.md

```diff
@@ -34,15 +34,15 @@ The format version number is incremented when new features are added that will b
 
 Version 1 of the Iceberg spec defines how to manage large analytic tables using immutable file formats: Parquet, Avro, and ORC.
 
-All version 1 data and metadata files are valid after upgrading a table to version 2. [Appendix E](#version-2) documents how to default version 2 fields when reading version 1 metadata.
+All version 1 data and metadata files are valid after upgrading a table to version 2. [Appendix E](spec/#version-2) documents how to default version 2 fields when reading version 1 metadata.
```

Comment:
@samredai, any idea why anchor links don't keep you on the same page for the spec?

Reply:
It's definitely an unexpected behavior. I'll take a closer look into this and see if there's something in the theme we can tweak.